pkg/scripts/environment.go

```diff
@@ -49,10 +54,10 @@ func GetEnvironment(config *api.Config) ([]Environment, error) {
 			Name:  strings.TrimSpace(parts[0]),
 			Value: strings.TrimSpace(parts[1]),
 		}
-		glog.V(1).Infof("Setting '%s' to '%s'", e.Name, e.Value)
 		result = append(result, e)
 	}
 
+	glog.Infof("Setting %d environment variables provided by '.s2i/environment'", len(result))
```

Comment:
yes, same concert... will change that to "provided by environment file in sources"